docs/modules/ROOT/pages/explanations/decisions/postgres-monitoring.adoc

```diff
@@ -0,0 +1,109 @@
+= PostgreSQL Managed by VSHN Monitoring
+
+== Problem
+
+We need to provide PostgreSQL Managed by VSHN monitoring checks:
+
+* Availability check
+
+== Requirements for availability check
+
+* PostgreSQL is accessible on network level
+* it's possible to login to PostgreSQL
+* it's possible to execute queries 
+
+== Solutions
+
+PostgreSQL managed by Stackgres:
+
+* it doesn't reuse generated connection string which our users will use, which could lead to situation where our monitoring is green, but users can't login (issues with Service)
+* it's sidecar, issue with exporter would require pod restarting
+* prometheusAutobind from SGCluster doesn't work, so I have a fear that whole monitoring isn't really well tested
+
+PostgreSQL managed by VSHN:
+
+* the same functionalities as above
+* solves issues presented above
+
+== Decision
+
+Use postgresql_exporter managed by VSHN, created as Kubernetes Object in our composition:
+```
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: exporter-testing
+  labels:
+    name: exporter-testing
+---
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNPostgreSQL
+metadata:
+  name: my-postgres-example
+  namespace: exporter-testing
+spec:
+  parameters:
+    service:
+      majorVersion: "15"
+    backup:
+      schedule: '0 22 * * *'
+    size: 
+      cpu: "300m"
+      memory: "64Mi"
+      disk: "1Gi"
+  writeConnectionSecretToRef:
+    name: postgres-creds-connection
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "postgres-exporter"
+  namespace: exporter-testing
+  labels:
+    app: "postgres-exporter"
+spec:
+  containers:
+  - name: postgres-exporter
+    image: "prometheuscommunity/postgres-exporter"
+    command: ["/bin/sh", '-c']
+    args: ["/bin/postgres_exporter --log.level=debug"]
+    resources:
+      limits:
+        cpu: 200m
+        memory: 500Mi
+      requests:
+        cpu: 100m
+        memory: 200Mi
+    env:
+    - name: DATA_SOURCE_URI
+      valueFrom:
+        secretKeyRef:
+          name: postgres-creds-connection
+          key: POSTGRESQL_HOST
+    - name: DATA_SOURCE_USER
+      valueFrom:
+        secretKeyRef:
+          name: postgres-creds-connection
+          key: POSTGRESQL_USER
+    - name: DATA_SOURCE_PASS
+      valueFrom:
+        secretKeyRef:
+          name: postgres-creds-connection
+          key: POSTGRESQL_PASSWORD
+    - name: PG_EXPORTER_AUTO_DISCOVER_DATABASES
+      value:  'true'
+    ports:
+    - containerPort:  9187
+  restartPolicy: Always
+---
+# kubectl -n exporter-testing port-forward pods/postgres-exporter 9187
+# curl localhost:9187/metrics | grep -v '#'
+```
```

Comment:
No technical details needed in a decision document this way. Please remove that again.